lib/connection.js

```diff
@@ -193,6 +197,10 @@ class Connection extends EventEmitter {
   _heartbeatCallback(interval) {
     this.transport.send('{}');
     this.setTimeout(interval, this._heartbeatCallbackInstance);
+
+    if (process.env.NODE_ENV !== 'production') {
+      this.emit('sentPacket', this._heartbeatMessage);
```

Comment:
Actually I think it'll be okay to test both behaviors based on current env variable value.